TSC-Charter.md

```diff
@@ -81,21 +81,31 @@ Subject to such policies as may be set by the Board, the TSC is
 responsible for all technical development within the Node.js Foundation,
 including:
 
-* Setting release dates.
-* Release quality standards.
-* Technical direction.
-* Project governance and process (including this policy).
-* GitHub repository hosting.
-* Conduct guidelines.
-* Maintaining the list of additional Collaborators.
-* Development process and any coding standards.
+* Setting release dates
+* Release quality standards
+* Technical direction
+* Maintaining the list of additional Collaborators
+* Development process and any coding standards
 * Mediating technical conflicts between Collaborators or Foundation
-projects.
+projects
+* Management of the `nodejs-private` GitHub organization
 
 The TSC will define Node.js Foundation’s release vehicles and serve as
 Node.js Foundation’s primary technical liaison body with external open
 source projects, consortiums and groups.
 
+The TSC shares responsibility with the Node.js Community Committee ("CommComm")
+for:
+
+* Management of the `nodejs` GitHub organization
+* Project governance and process
```

Comment:
I will need to think about this as it might be a bit too broad. I think the TSC should retain full control over processes that are technical in nature (security, releases, etc).

Reply:
@mcollina would `Non-technical project governance and process` work for you?

Reply:
I would prefer a positive definition, but that would work.

Reply:
I'm ok with the current wording. In fact, this is already present in the https://github.com/nodejs/community-committee/blob/master/Community-Committee-Charter.md charter. I'm not 100% sure how this could have been in the commcomm charter and here as well at the same time.

I think the commcomm charter would need to be amended about the shared responsibility.